src/platforms/unity/index.mdx

```diff
@@ -12,21 +17,99 @@ If you don't already have an account and Sentry project established, head over t
 
-<PlatformContent includePath="getting-started-install" />
+</Note>
 
 ## Configure
 
-Configuration should happen as early as possible in your application's lifecycle.
+Configuration should happen as early as possible in your application's lifecycle. Currently, there is only one approach to configure Sentry - Unity tools window.
+
+Open `Tools` -> `Sentry`. 
+
+The first time you open the window, needed Unity Sentry SDK related files will be created: 
+
+* `Assets/Resources/Sentry/SentryOptions.json` - Sentry settings
+* `Assets/Resources/Sentry/link.xml` - [a special file](https://docs.unity3d.com/Manual/ManagedCodeStripping.html) for [ILL2CPP](https://docs.unity3d.com/Manual/IL2CPP.html) builds
+
+![Sentry tools](tools_sentry.png)
+
+Sentry configuration window will be opened.
+
+![Sentry window](sentry_window.png)
 
-<PlatformContent includePath="getting-started-config" />
+* `Enable` - enable Sentry events capturing
+* `DSN` - Sentry DSN value
+
+After you close the window, all settings will be saved into `Assets/Resources/Sentry/SentryOptions.json`.
 
 ## Verify
 
-This snippet includes an intentional error, so you can test that everything is working as soon as you set it up:
+Create `TestMonoBehaviour.cs` which contains intentional error (in this case `Debug.LogError`, but can be any exception):
+
+```csharp
+using UnityEngine;
+
+public class TestMonoBehaviour : MonoBehaviour
+{
+    void Start()
+    {
+        Debug.Log("Log");
+        Debug.LogWarning("Warning");
+        Debug.LogError("Error");
+    }
+}
+```
+
+Assign it to any object you have in the scene and hit play. Open up your Sentry project in web UI and check the logs. 
+
+You should see a new issue:
+
+![Sentry issue](sentry_issue.png)
+
+Open it up and check breadcrumbs:
+
+![Sentry breadcrumbs](sentry_breadcrumbs.png)
+
+The list of captured events are displayed.
+
+### Event debouncer
+
+There is a debouncing mechanism in place. It means that if you try to send a bunch of requests in a quick succession (with the same log type), we will take the first one and ignore the other ones given some time slot.
+
+Current settings:
+
+* `LogType.Error` (also exceptions) or `LogType.Exception` or `LogType.Assert` - 1 second
+* `LogType.Log` - 1 second
+* `LogType.Warning` - 1 second
+
+Create another `DebouncerTestMonoBehaviour.cs` assign to any object in the scene and run it.
+
+```csharp
+using UnityEngine;
 
-<PlatformContent includePath="getting-started-verify" />
+public class DebouncerTestMonoBehaviour : MonoBehaviour
+{
+    void Start()
+    {
+        Debug.Log("Log");             // recorded
+        Debug.Log("Log2");            // not recorded (1 second didn't expire from the previous call)
+        Debug.LogWarning("Warning");  // recorded
+        Debug.LogWarning("Warning2"); // not recorded
+        Debug.LogError("Error");      // recorded
+    }
+}
+```
 
 <Note>
```

Comment:
We usually have some "Next Steps" here, to help the user learn about Sentry. @PeloWriter might have ideas

Reply:
We'll do this on follow up PRs particularly because there's no way to opt-out of the Sentry Window init yet, so those options that can only set via SentryOptions callback can't be used yet anyway